src/engine.ts

```diff
@@ -29,6 +33,12 @@ export default class Engine {
   roundBoosters:  {
     [key in Booster]?: boolean 
   } = { }; 
+  techTiles: {
+    [key in TechTilePos]?: {tile: TechTile; available: boolean}
```

Comment:
instead of `{available: boolean}` it's something like `count: number`, since there are several of each tile

Reply:
correct, my bad